containers/n8n/n8n.bc.yaml

```diff
@@ -44,14 +44,18 @@ objects:
           ARG N8N_VERSION
           ENV HOME=/opt/app-root/src \
               TZ=America/Vancouver
+          USER root
```

Comment:
Would it be useful to move this whole dockerfile definition into its own file, rather than writing it inline here?

Reply:
It is easier to iterate settings this way, you can do it all on openshift. That was the only reason.
The dockerfile for this is not useful for anything but openshift though. For local we'd just use n8n/n8n:latest.